usr/local/share/bastille/bootstrap.sh

```diff
@@ -336,7 +336,7 @@ bootstrap_template() {
             $(which git) clone "${_url}" "${_template}" ||\
                 error_notify "Clone unsuccessful."
         elif [ -d "${_template}/.git" ]; then
-            cd "${_template}" && $(which git) pull ||\
+            (cd "${_template}" && $(which git) pull) ||\
```

Comment:
More of a "previous code" question than about your change: why are we doing `$(which git)` here (and 4 times)? If that finds _git_, then it would be found by simply calling `git`. Seems like an unnecessary subshell. The first time it's used makes sense to me since `-x` needs to know the path to the file, but the second time could be an `else` and the other 2 times could use `-C`.

On the change itself, it also seems like an unnecessary subshell. Git can be ran in any directory with `-C`. I believe all we need is this:
```
git -C "${_template}" pull
```

Reply:
Actually, it's better to say: `if ! which -s git;` instead `if [ ! -x "$(which git)" ];`
(-s means silent)